pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -21,6 +23,8 @@ import (
 	"kubevirt.io/client-go/log"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/api"
 	"kubevirt.io/kubevirt/pkg/virt-launcher/virtwrap/cli"
+
+	"github.com/go-xmlfmt/xmlfmt"
```

Comment:
I don't think that you need this library.
You can use something like:

```
	dom := &libvirtxml.Domain{}
	_ = xml.Unmarshal([]byte(domain), &dom)
	buf, _ := xml.MarshalIndent(dom, "", "\t")
	fmt.Println(string(buf))
```

and then we use the official xml library and libvirt's xml library.

Reply:
@danielBelenky Thanks for the advice! I've updated the PR